### ISSUE TITLE
Website issues

### ISSUE THREAD

**Issue by ljacqu** [brianleroux/wtfjs#147]
Great to see the site back up again—I'd like to report a few issues I've noticed while browsing the site.

- The first entry on https://wtfjs.com/, "2016 03 19 wierd instanceof", leads to a page that just says "wtf! not found"

- You're not setting the charset to UTF-8. E.g. https://wtfjs.com/wtfs/2014-02-20-math-pow shows "â€” @123jimin" instead of "— @123jimin" in my browser

- https://wtfjs.com/wtfs/2013-12-15-charAt-is-not-the-same-as-[] is just an empty page

- https://wtfjs.com/wtfs/2010-04-31-isNaN links to https://wtfjs.com/2010/02/12/null-is-not-an-object which just shows `{"message":"Missing Authentication Token"}` The link should be https://wtfjs.com/wtfs/2010-02-12-null-is-not-an-object

- https://wtfjs.com/wtfs/2015-04-23-screwy-negative-array-index is missing newlines

**Comment by DTrejo** [comment]
blerg :/

@ljacqu would you be open to helping with a pull request to fix these? I'm not asking you to make one right now, I'm just curious if you're be open to helping.

**Comment by ljacqu** [comment]
Absolutely! Some of them I don't think I can help, but I can definitely contribute some. I'll open a pull request tomorrow.

**Comment by DTrejo** [comment]
wahoo! :)

**Comment by ljacqu** [comment]
I won't be able to submit a PR today but I've started doing some copy-editing. At a later time I'll get the website running locally and I'll try to take care of more technical stuff, such as the encoding issue. Cheers :)

**Comment by DTrejo** [comment]
Sounds good - go easy on yourself :)

**Comment by ljacqu** [comment]
Thanks! A pull request will follow on Sunday that will address the points in the issue description ;3

**Comment by DTrejo** [comment]
Cool :) even if you only fix one, still send the PR!

**Comment by shaedrich** [comment]
Looks like everythink except your third bullet point have been addressed in the now merged PR. Furtunately, this has been addressed in #160
